reya_data_feed/resources/market.py

```diff
@@ -0,0 +1,164 @@
+"""Market-related WebSocket resources."""
+from typing import Dict, Optional, Any
+from .common import WebSocketResource, ParameterizedResource, SubscribableResource, SubscribableParameterizedResource
+
+class MarketResource:
+    """Container for all market-related WebSocket resources."""
+    
+    def __init__(self, socket: 'ReyaSocket'):
+        """Initialize the market resource container.
+        
+        Args:
+            socket: The WebSocket connection to use for this resource.
+        """
+        self.socket = socket
+        self._all_markets = AllMarketsResource(socket)
+        self._market_data = MarketDataResource(socket)
+        self._market_orders = MarketOrdersResource(socket)
+    
+    @property
+    def all_markets(self) -> 'AllMarketsResource':
+        """Access the all markets data resource."""
+        return self._all_markets
+    
+    def market_data(self, market_id: str) -> 'MarketDataSubscription':
```

Comment:
i think we should also have a separate channel for market prices
currently we have an endpoint that surfaces prices for both markets and collateral assets
i think it'd be nice to have a separate market level endpoint for prices and within that channel we expose both the oracle price (mark price) and the pool price (mid price)

within that i think it'd be good to clean up the terminology, terms like oracle price and pool price are not very standard, think makes me think that it'd be nice to setup a quick notion ticket where we align on all the terminology and will be nice to get @aaraujoxr 's input on that as well

Reply:
think we can also bring candle streams as well, i believe we already have them but don't think they are under the market resource, need to double check

Reply:
As discussed, added missing websocket channel for prices and all the other trading API REST endpoints !